spec/factories/goals.rb

```diff
@@ -1,15 +1,15 @@
 FactoryGirl.define do
   factory :goal do
-    user
+    # user
```

Comment:
I agree, however this functionality is currently not present within bepstore-api. So that should be fixed first.